### PR TITLE
Add k8s cassandra use case

### PR DESCRIPTION
**Pull request by avestuk**
Adds a use case for Cassandra.

### CODE REVIEW

**Comment by actionbuddha** [comment]
@avestuk Can we include a docker version plz? I want to ensure we retain parity between the two mechanisms.

**Comment by avestuk** [comment]
Docker use case added.

File: _docs/usecases/docker/cassandra.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+   storageos           cassandra-data3
+   ```
+
+1. Run a Cassandra container using the StorageOS volume driver.
+
+   ```bash
+   $ docker run -d --name cassandra-docker \
+   -v cassandra-data:/var/lib/cassandra    \
+   cassandra:3.11
+   e701e4945fb6730127ecfce8459faad5d6a08fbd53dd6f68f336ff8bf092d470
```
Comment (actionbuddha) [inline _docs/usecases/docker/cassandra.md]: What's the purpose of this GUID here?

Comment (avestuk) [reply _docs/usecases/docker/cassandra.md]: Just an example of the output you'd expect from the command. I can see how it's a little misleading as others won't get that GUID. I can remove it.

Comment (avestuk) [reply _docs/usecases/docker/cassandra.md]: It's been removed

**Pull request merged**